### PR TITLE
docs (workflows): 2D embeddings

### PR DESCRIPTION
**Pull request by Gorkem-Encord**

### CODE REVIEW

File: docs/docs/workflows/using-2d-embeddings.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 3
+---
+
+# Using 2D Embeddings
+
+**Use 2D embeddings to leverage your active learning cycle**
```
Comment (frederik-encord) [inline docs/docs/workflows/using-2d-embeddings.md:7]:
```suggestion
**Use 2D embeddings to improve your active learning cycle**
```

File: docs/docs/workflows/using-2d-embeddings.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Encord Active presents you 2D embedding plot for all the images in your project. You can use it to find interesting
```
Comment (frederik-encord) [inline docs/docs/workflows/using-2d-embeddings.md:9]:
```suggestion
Encord Active presents you with a 2D embedding plot for all the images in your project. You can use it to find interesting 
```

File: docs/docs/workflows/using-2d-embeddings.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 3
+---
+
+# Using 2D Embeddings
+
+**Use 2D embeddings to leverage your active learning cycle**
+
+Encord Active presents you 2D embedding plot for all the images in your project. You can use it to find interesting 
+clusters, understand your data better, weakly label your images, remove detrimental images and many more. 2D
```
Comment (frederik-encord) [inline docs/docs/workflows/using-2d-embeddings.md:10]:
```suggestion
clusters, understand your data better, weakly label your images, remove detrimental images, etc. The 2D 
```

File: docs/docs/workflows/using-2d-embeddings.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+sidebar_position: 3
+---
+
+# Using 2D Embeddings
+
+**Use 2D embeddings to leverage your active learning cycle**
+
+Encord Active presents you 2D embedding plot for all the images in your project. You can use it to find interesting 
+clusters, understand your data better, weakly label your images, remove detrimental images and many more. 2D 
+embedding plot can be visualized in the **Explorer** pages of the **Data Quality** and **Label Quality** sections. Here is an example
```
Comment (frederik-encord) [inline docs/docs/workflows/using-2d-embeddings.md:11]:
```suggestion
embedding plot can be found in the **Explorer** pages of the **Data Quality** and **Label Quality** sections. Here is an example
```

File: docs/docs/workflows/using-2d-embeddings.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+embedding plot for the COCO 2017 validation set.
+
+![using-2d-embeddings-1](../images/workflows/using-2d-embeddings/using-2d-embeddings-1.png)
+
+You see how images can be clustered around certain regions. In order to visualize images in these clusters and learn
```
Comment (frederik-encord) [inline docs/docs/workflows/using-2d-embeddings.md:16]:
```suggestion
Notice how images are clustered around certain regions. In order to visualize images in these clusters and learn
```

File: docs/docs/workflows/using-2d-embeddings.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+what is common among these samples, you can use Box or Lasso Select on the upper right corner of the plot.
```
Comment (frederik-encord) [inline docs/docs/workflows/using-2d-embeddings.md:17]:
```suggestion
what is common among these samples, you can use the _Box_ or _Lasso Select_ in the upper right corner of the plot.
```

File: docs/docs/workflows/using-2d-embeddings.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+snowy images.
+
+![img.png](../images/workflows/using-2d-embeddings/using-2d-embeddings-3.png)
+
+Now you can perform several actions with the selected group:
+- You can tag and export them in the **Actions** => **Filter & Export** page so that you can label these images 
+automatically via a script.
+- If you do not want them in your dataset for some reason, you can tag all the images and remove the tags from the
+selected ones so that you can create a new version of your dataset without them. You can specifically apply this step
+for the images that you think including them will be detrimental.
```
Comment (frederik-encord) [inline docs/docs/workflows/using-2d-embeddings.md:32]: I wouldn't mention the deleting part. I would probably make a less descriptive enumeration of
1. Data exploration
2. Soft labelling via tags
3. Tag subsets for labelling

File: docs/docs/workflows/using-2d-embeddings.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+Samples in the 2D embedding plot in the **Data Quality => Summary** page have no label information; therefore, they are all in
```
Comment (frederik-encord) [inline docs/docs/workflows/using-2d-embeddings.md:34]:
```suggestion
Samples in the 2D embedding plot in the **Data Quality => Summary** page have no label information; therefore, they all share
```

File: docs/docs/workflows/using-2d-embeddings.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+the same color. On the other hand, it is colored according to labels on the **Label Quality => Summary** page.
```
Comment (frederik-encord) [inline docs/docs/workflows/using-2d-embeddings.md:35]:
```suggestion
the same color. On the other hand, points colored according to labels on the **Label Quality => Summary** page. 
```

File: docs/docs/workflows/using-2d-embeddings.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+Samples in the 2D embedding plot in the **Data Quality => Summary** page have no label information; therefore, they are all in
+the same color. On the other hand, it is colored according to labels on the **Label Quality => Summary** page. 
+
+![using-2d-embeddings-4](../images/workflows/using-2d-embeddings/using-2d-embeddings-4.png)
+
+Now you can:
+- check which classes are confused with each other
+- spot wrongly labeled samples (e.g., a different class inside the large cluster of another class)
+- detect outliers and remove them from the dataset
```
Comment (frederik-encord) [inline docs/docs/workflows/using-2d-embeddings.md:42]:
```suggestion
- Check which classes are confused with each other
- Spot wrongly labeled samples (e.g., a different class inside the large cluster of another class)
- Detect outliers and remove them from the dataset
```

File: frontend_components/src/encord_active_components/__init__.py [hunk 11]
```diff
@@ -20,8 +20,8 @@ class MenuItem(TypedDict):
     _pages_menu = components.declare_component("pages-menu", url="http://localhost:5173")
 
 
-def pages_menu(items: List[MenuItem]) -> str:
-    return _pages_menu(items=items)
+def pages_menu(menu_items: List[MenuItem]) -> str:
+    return _pages_menu(items=menu_items)
```
Comment (frederik-encord) [inline frontend_components/src/encord_active_components/__init__.py:24]: This seems unrelated to this PR?

**Review by frederik-encord — CHANGES_REQUESTED** [review]

File: docs/docs/workflows/using-2d-embeddings.md (hunk 7 above)
Comment (Gorkem-Encord) [reply docs/docs/workflows/using-2d-embeddings.md:32]: Did not quite understand the difference between 2 and 3. Wrote sth, feel free to change.

File: frontend_components/src/encord_active_components/__init__.py (hunk 11 above)
Comment (Gorkem-Encord) [reply frontend_components/src/encord_active_components/__init__.py:24]: 😅 definitely.

**Review by frederik-encord — APPROVED** [review]
